model_lab_configs/huggingface/Intel/bert-base-uncased-mrpc/1/bert_ov.json.config

```diff
@@ -1,6 +1,7 @@
 {
     "name": "Convert to Intel NPU",
     "oliveFile": "bert/openvino/bert_base_uncased_mrpc/bert-base-uncased-mrpc_context_ov_static.json",
+    "isIntel": true,
```

Comment:
do we need this? In my understanding, it should be a general that
`runtime` -> Runtime for evaluation
`runtimeInConversion` -> conversion